self-run-agents/instances/azp-build-asg/instances.tf

```diff
@@ -107,48 +100,12 @@ resource "aws_autoscaling_group" "build_pool" {
     }
   ]
 
-  lifecycle {
-    ignore_changes = ["desired_capacity"]
-  }
-
   # use1-az6, use1-az2, use1-az4
-  # The ones with a1.4xl's.
-  vpc_zone_identifier = ["subnet-29a65576", "subnet-33e41912", "subnet-b88b0df5"]
-}
-
-# Can't use the recommend initial_lifecycle_hook due to:
-# https://github.com/terraform-providers/terraform-provider-aws/issues/9841
-#
-# This will probably never be a problem since the only window would be between
-# ASG Creation + Lifecycle Hook Creation which should be immediate, and doubtful
-# we'll terminate any instances then.
-#
-# If we do we can do a one off delete.
-resource "aws_autoscaling_lifecycle_hook" "lifecycle_hook" {
-  name                    = "${var.ami_prefix}_${var.azp_pool_name}_azure_dereg_hook"
-  default_result          = "CONTINUE"
-  heartbeat_timeout       = 2000
-  lifecycle_transition    = "autoscaling:EC2_INSTANCE_TERMINATING"
-  notification_target_arn = var.sns_lifecycle_arn
-  role_arn                = var.sns_lifecyle_role_arn
-
-  autoscaling_group_name = aws_autoscaling_group.build_pool.name
-}
-
-resource "aws_autoscaling_policy" "nodes-on-demand" {
-  autoscaling_group_name = aws_autoscaling_group.build_pool.name
-  name                   = "${var.ami_prefix}_${var.azp_pool_name}_tt"
-  policy_type            = "TargetTrackingScaling"
-
-  target_tracking_configuration {
-    predefined_metric_specification {
-      predefined_metric_type = "ASGAverageCPUUtilization"
-    }
-
-    # Someone shold probably apply Queue'ing theory to this to make it
-    # actually a reasonable number, for now 60 seems like it'll work
-    # since builds take 100% CPU, so this means more than half of our
-    # workers are busy.
-    target_value = 60.0
-  }
+  # The ones with r6g.4xl's.
+  vpc_zone_identifier = [
+    "subnet-224abd44", # us-east-1a
```

Comment:
nit: Zone `1a` isn't guaranteed to be the same in every account. Can we switch back to the availability zone numbers to get a consistent idea of what AZs we're in?

Reply:
done